app/domain/authentication/authn_jwt/signing_key/public_signing_keys.rb

```diff
@@ -0,0 +1,42 @@
+
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # This class is a POJO class presents public-keys structure
+      class PublicSigningKeys
+        include ActiveModel::Validations, AttrRequired
+
+        VALID_TYPES =  %w(jwks).freeze
```

Comment:
Maybe move this to consts?

Reply:
As far as the class is the only consumer of the value it feels me unnecessary